src/snowflake/snowpark/modin/plugin/utils/numpy_to_pandas.py

```diff
@@ -195,46 +195,46 @@ def map_to_bools(inputs: Any) -> Any:
     "logaddexp2": NotImplemented,
     "true_divide": lambda obj, inputs: obj.__truediv__(*inputs),
     "floor_divide": lambda obj, inputs: obj.__floordiv__(*inputs),
-    "negative": NotImplemented,
-    "positive": NotImplemented,
+    "negative": lambda obj, inputs: -obj,
+    "positive": lambda obj, inputs: obj,
     "power": NotImplemented,  # cannot use obj.__pow__
     "float_power": lambda obj, inputs: obj.__pow__(*inputs),
     "remainder": lambda obj, inputs: obj.__mod__(*inputs),
     "mod": lambda obj, inputs: obj.__mod__(*inputs),
     "fmod": NotImplemented,
     "divmod": NotImplemented,
-    "absolute": NotImplemented,
-    "fabs": NotImplemented,
+    "absolute": lambda obj, inputs: obj.abs(),
+    "abs": lambda obj, inputs: obj.abs(),
     "rint": NotImplemented,
     "sign": NotImplemented,
     "heaviside": NotImplemented,  # heaviside step function
     "conj": NotImplemented,  # same as conjugate
     "conjugate": NotImplemented,
-    "exp": NotImplemented,
+    "exp": lambda obj, inputs: obj.apply(sp_func.exp),
     "exp2": NotImplemented,
     "log": lambda obj, inputs: obj.apply(sp_func.ln),  # use built-in function
```

Comment:
nit: looks like this comment of `# use built-in function` stands out a bit odd comparing to many other usage of built-in funciton

Reply:
+1